lib/class-wp-rest-menu-items-controller.php

```diff
@@ -959,10 +919,11 @@ public function get_item_schema() {
 			'description' => __( 'The DB ID of the nav_menu_item that is this item\'s menu parent, if any, otherwise 0.', 'gutenberg' ),
 			'context'     => array( 'view', 'edit', 'embed' ),
 			'type'        => 'integer',
-			'minimum'     => 0,
-			'default'     => 0,
+			'minimum'     => 1,
```

Comment:
This means, that validation, triggers validation.

Reply:
Are there _any_ cases where this can be `0`? You should generally be able to `PUT` back a `GET` response without any errors.

Reply:
Original discussion is here https://github.com/WordPress/gutenberg/pull/34672#discussion_r706133141

Reply:
Ok, let's leave as is for now.